### DISCUSSION TITLE
LTSP Login not working after sleep

### DISCUSSION THREAD

**Discussion by keerthi-varuman** [ltsp/ltsp#741] [General] [answered]
I am using Ubuntu 20.04 LTS.
I can not login to the user account after sleep mode it is saying that the user account or password is wrong. 
I am using lightdm display manager

**Comment by alkisg** [comment]
If you mean "on the LTSP server", then that's not an LTSP issue, you may report it to your distribution bug tracker.

If you mean "on LTSP clients", sleep mode isn't supported there, it should be disabled.

**Reply by keerthi-varuman** [reply]
Thank you and I have disabled the sleep mode on the LTSP client. when I do the lock screen and try to login back into the user account it is saying that the user name or password does not exist. and i need to do switch user to login again

**Comment by alkisg** [comment]
Which Ubuntu flavor is that?
Also post a photo of the lock screen dialog,
and the output of the following command (to see which screensaver you're using):

ps faux | grep screen

**Comment by keerthi-varuman** [comment]
I am using Ubuntu 20.04 LTS server edition.
I have installed `ubuntu-unity-desktop` with `lightdm` display manager.
I never set up any screen saver when I go out I just press windows + L to lock the screen that's it.  
when I want to login to my user again from the lock screen. I am getting the message that the user name or password is wrong.

Here is the output of ps faux | grep screen
`root@entry:~# ps faux | grep screen
root       51086  0.0  0.0   6440   720 pts/0    S+   15:24   0:00          \_ grep --color=auto screen
suvedha    51056  0.0  0.9 384756 36816 ?        Ssl  15:22   0:00  \_ /usr/lib/x86_64-linux-gnu/unity/unity-panel-service --lockscreen-mode
`

**Comment by alkisg** [comment, marked as answer]
That's a custom setup, AFAIK unity isn't used in any of the flavors anymore.
Sorry but I don't want to spend time troubleshooting this custom setup! 
It would be much saner to use one of the official flavors, MATE, KDE, GNOME, using their official .iso for installation.

**Reply by keerthi-varuman** [reply]
I have installed gnome and gdm display manager and the problem is solved
